app/feeds/new/add-operator/template.hbs

```diff
@@ -11,6 +12,11 @@
 			{{documentation-link "add-feed.md#choose-the-transit-operators-in-the-feed" "Learn more about transit operators"}}
 		</p>
 	</div>
+	{{#if feedExists}}
+		<div class="feed-exists-warning">
+			It looks like this feed already exists in the datastore! Please add to or edit the information we currently have.
```

Comment:
"Looks like this feed already exists in the Transitland. Please review and edit the information below."